pkg/components/cloudcredentialoperator/component.go

```diff
@@ -21,6 +21,8 @@ var CloudCredentialOperatorComponent = Component{
 			{
 				IncludeAll: []string{"bz-Cloud Credential Operator"},
 			},
+			{Suite: "CloudCredentialOperator components upgrade tests"},
+			{Suite: "cloud credential operator"},
```

Comment:
For below golang test case as example, it won't be mapping into the Suites? 
g.It("NonHyperShiftHOST-ROSA-OSD_CCS-Author:jshu-LEVEL0-Critical-36498-CCO credentials secret change to STS-style", func()